config/coaddDriver.py

```diff
@@ -3,6 +3,12 @@
 
 from lsst.utils import getPackageDir
 
+from lsst.pipe.tasks.assembleCoadd import CompareWarpAssembleCoaddTask
+config.assembleCoadd.retarget(CompareWarpAssembleCoaddTask)
+config.assembleCoadd.assembleStaticSkyModel.subregionSize = (10000, 200)
+config.assembleCoadd.assembleStaticSkyModel.badMaskPlanes = ("BAD", "EDGE", "SAT", "INTRP", "NO_DATA",)
+config.assembleCoadd.assembleStaticSkyModel.doApplyUberCal = True
```

Comment:
To eliminate code duplication, this should load the configuration from the configuration file for compareWarpAssembleCoadd (like below). And I think the additional loading below may potentially clobber what you've got here.

Reply:
Agreed.  Note that by that reasoning it currently should be reading in safeClipAssembleCoadd.py

Reply:
I think https://jira.lsstcorp.org/browse/DM-10634 might be relevant here.